spec/system/planning_applications/review/sign_off_spec.rb

```diff
@@ -262,7 +262,7 @@
       click_button "Save"
 
       within(".govuk-form-group--error") do
-        expect(page).to have_content("Please state the reasons why this application is, or is not lawful")
+        expect(page).to have_content("Please State the reasons for your recommendation.")
```

Comment:
`state` and not `State`

Reply:
Missed it as this spec never run as is marked as `flaky`